src/vm/excep.cpp

```diff
@@ -7280,8 +7280,17 @@ bool IsIPInMarkedJitHelper(UINT_PTR uControlPc)
 {
     LIMITED_METHOD_CONTRACT;
 
+#ifdef _ARM_
+    // The least bit denoting THUMB might confuse the range check because
+    // PC values have the least bit reset. It causes a bug if PC is at
+    // the first byte of the function range.
+#define CHECK_RANGE(name) \
+    if ((GetEEFuncEntryPoint(name)|0x1) <= (uControlPc|0x1) && (uControlPc|0x1) < (GetEEFuncEntryPoint(name##_End)|0x1)) return true;
```

Comment:
I would guess it is the case. @jkotas, am I right? Does the GetEEFuncEntryPoint on ARM always return address with the bit 0 set?